doc/source/cluster/kubernetes/examples/rayjob-kueue-gang-scheduling.md

```diff
@@ -155,6 +156,12 @@ kubectl apply -f kueue-resources.yaml
 This example configures Kueue to orchestrate the gang scheduling of GPUs. However, you can use other resources such as CPU and memory.
 :::
 
+:::{note}
+Google Kubernetes Engine's queued provisioning feature currently supports only single PodSet per request. To circumvent this issue, we
+set `podSetMergePolicy: IdenticalWorkloadSchedulingRequirements` in the `ProvisioningRequestConfig`. When giving the head node and the
+worker nodes the same resource requirements, affinities, and tolerations, Kueue merges them into a single PodSet in the `ProvisioningRequest`.
```

Comment:
For some additional context, the example only uses a single head node because adding multiple workers would require setting up a shared volume for distributed checkpointing. If you add the shared volume and the head pod is not requesting GPUs, it shouldn't need podSetMergePolicy, but let me know if you see different behavior

Reply:
Indeed if "head" is using CPU this should not be needed if user can use "managedResources" as also summarized in https://github.com/ray-project/ray/pull/59070#issuecomment-3638211104.

In not sure we need to mention on that detail here since we refer to Kueue's documentation.